### PR TITLE
- subtitle not constrained to show the amount of data 

### PR DESCRIPTION
**Pull request by mo0om**
Hi, I think the subtitle should be more flexible this way. Here is a use case where the subtitle as it is is not working for us...
UX want me to display in the sub title...
There are 5 tenants, 30 assets group and 2 users
but because the way it is set now I'm getting...
5 - There are 5 tenants, 30 assets group and 2 users

Since it's easy to get this value from the model I don't see why we have to restrain people to format the subtitle like this...** {num} - "the string"**
#minor#

### CODE REVIEW

File: addon/templates/components/frost-object-browser.hbs [hunk 1]
```diff
@@ -4,7 +4,9 @@
       {{title}}
     </div>
     {{# if subtitle}}
-     <div class='sub-title'>{{computedValuesTotal}} - {{subtitle}}</div>
+     <div class='sub-title'>
+         {{#if showCountInSubTitle}}{{computedValuesTotal}} -{{/if}} {{subtitle}}
```
Comment (sandersky) [inline addon/templates/components/frost-object-browser.hbs]: I think it is generally bad practice to use helpers in templates for conditionals as it is hard to test. Instead I would use a computed property like so.

**Component:**
```javascript
@readOnly
@computed('subtitle', 'showCountInSubTitle', 'computedValuesTotal')
computedSubtitle (subtitle, showCountInSubTitle, count) {
  return showCountInSubTitle ? `${count} – ${subtitle}` : subtitle
}
```

**Template:**
```handlebars
<div class="sub-title">
  {{computedSubtitle}}
</div>
```

Then you can write unit tests against this computed property given different values for the three properties it is computed off of.

Comment (mo0om) [inline addon/templates/components/frost-object-browser.hbs]: yes, it true it looks cleaner on the template. 
Should I do another pull request?
